clients/src/main/java/org/apache/kafka/common/record/FileRecords.java

```diff
@@ -108,14 +108,17 @@ public FileChannel channel() {
     }
 
     /**
-     * Read log entries into a given buffer.
-     * @param buffer The buffer to write the entries to
-     * @param position Position in the buffer to read from
-     * @return The same buffer
-     * @throws IOException
+     * Read data from the channel to the given byte buffer until there are no bytes remaining in the buffer or
+     * the channel has reached end of stream
```

Comment:
sorry, but I do not catch your meaning. Do you mean we should modify the comments for readInto method to explain why we use readFully instead of naive Channel.read?